pkg/ovs/openflow/ofctrl_priority.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+type Priority uint16
+
+func NewPriority(val uint16) Priority {
+	return Priority(val)
+}
+
+func (p Priority) Add(offset uint16) Priority {
+	return Priority(uint16(p) + offset)
+}
+
+func (p Priority) Inc() Priority {
+	return Priority(uint16(p) + 1)
+}
```

Comment:
no strong opinion, but I notice the `Inc()` only being called once, maybe we can just keep `Add` to do the same thing.

Reply:
Thanks for reviewing this PR first!

Yes, currently `Inc()` is only called once, but in the future, I think it can be used more than once. IMO, I recommend using `Inc()` in pipeline design except traceflow since it is like a controllable extend of current priorities (like high, normal, low -> high + 1, high, normal + 1, normal, low + 1, low).  For `Add`, I recommend using it in traceflow design only.

Reply:
Also feel `Inc` is not necessary since `Add` could replace it.